src/components/common/successModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Please change the name of this file to SuccessModal

Reply:
Done